src/plugins/intel_gna/src/gna_plugin_internal.hpp

```diff
@@ -40,7 +40,13 @@ class GNAPluginInternal : public InferenceEngine::IInferencePlugin {
         const std::map<std::string, std::string>& config) override {
         std::lock_guard<std::mutex> lock{syncCallsToLoadExeNetworkImpl};
         Config updated_config(defaultConfig);
-        updated_config.UpdateFromMap(config);
+
+        auto tmp = config;
+        auto cache_dir = GetCore()->GetConfig(GetName(), ov::cache_dir.name());
+        if (cache_dir != ov::Any(""))
+            tmp[ov::cache_dir.name()] = cache_dir.as<std::string>();
+        updated_config.UpdateFromMap(tmp);
```

Comment:
why do we need to handle cache_dir in GNA explicitly? removed here https://github.com/openvinotoolkit/openvino/pull/16134